### DISCUSSION TITLE
Enabling all 8 SFPs on Alveo U250

### DISCUSSION THREAD

**Discussion by sohun-arches** [corundum/corundum#188] [Q&A] [answered]
I am looking at using Corundum as a networking diagnostic tool to check connections on remote machines with Alveo U250 cards installed. We use all 8 SFPs and want to have an implementation of Corundum with all 8 SFPs. Is this possible?

**Comment by alexforencich** [comment, marked as answer]
This is currently supported in my development repo.  It's not in the mainline repo quite yet, but it will be there in the near future.
